### ISSUE TITLE
Debugger crashed when debugging a Blazor WASM app in VS 16.6

### ISSUE THREAD

**Issue by mkArtak** [mono/mono#19811]
The Debugger crashed:

After successfully hitting a breakpoint and stepping through it - when the debugger was about to continue the execution of the app, it froze for a moment - like nothing is happening and then it crashed:
![image](https://user-images.githubusercontent.com/7347708/82414811-762b2900-9a2c-11ea-963d-48500494fb3d.png)

## Trying to repro here
1. The breakpoint hits successfully
![image](https://user-images.githubusercontent.com/7347708/82414959-af639900-9a2c-11ea-9bf5-fa5efa17c0ad.png)

2. So far so good
![image](https://user-images.githubusercontent.com/7347708/82415064-d1f5b200-9a2c-11ea-9c26-7882a036a51b.png)

3. F10 to continue causes this:
![image](https://user-images.githubusercontent.com/7347708/82415131-e8037280-9a2c-11ea-99dd-7c2afd9a8791.png)

4. This is taking too long
![image](https://user-images.githubusercontent.com/7347708/82415213-0ec1a900-9a2d-11ea-8e79-c9d5ede75e63.png)

5. After waiting for about 10 minutes I had to cancel it and it resulted in the same error:
![image](https://user-images.githubusercontent.com/7347708/82416019-3e24e580-9a2e-11ea-9d53-9c5abc038d5d.png)

Let me know what information I can provide to help with investigation.

**Comment by lewing** [comment]
Can you share the code to reproduce this?

**Comment by mkArtak** [comment]
I can repro this with a new Blazor WebAssembly app:
1. Install .NET Core 3.1.300 SDK
2. dotnet new bazorwasm
3. Open the project in VS 16.6
4. Open the `Counter.razor` component
5. Add a breakpoint in the `IncrememntCount()` method as follows:
![image](https://user-images.githubusercontent.com/7347708/82510710-1da56b80-9ac0-11ea-9b68-77ae66a0191b.png)
6. F5
7. Go to the counter page and click the `Click Me` button
8. Breakpoint will hit
9. F10 couple of times to get out of the method
10. VS hangs as shown in the earlier comments.

**Comment by radical** [comment]
I'm not able to reproduce this with the steps from the last comment. I'm using the same versions as mentioned there. Would you be able to record a video? Maybe I'm missing something subtle here.

**Comment by radical** [comment]
Could you also share text from the output window, and `C:\Users\<username>\AppData\Local\Temp\vscode-edge-debug.txt `, or `*-chrome-debug.txt`?

**Comment by mkArtak** [comment]
@radical, here is the file: https://gist.github.com/mkArtak/657a111cfc87fd71658267b44f304526

**Comment by mkArtak** [comment]
Here is the requested video.

![image](https://user-images.githubusercontent.com/7347708/82526194-3a559980-9ae8-11ea-80c3-cf3d9d36bc88.png)

**Comment by radical** [comment]
Which browser are you using? And version?

**Comment by radical** [comment]
I can reproduce this with Edge Dev.

**Comment by mkArtak** [comment]
@radical you're right, trying non-Dev Edge, I can't repro this. So there is some issue in the `beta` channel.
